website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+
+#### Compile button
+The Compile Button takes the contents of the active File Editor (regardless of save status) and returns the compiled SQL to the Compiled Code Subtab
+
+#### Build button 
+The build button is a shortcut for the user to invoke dbt commands anchored by the active model in the File Editor. The available commands are dbt build, dbt test, and dbt run with options to include only the current resource, the resource and its upstream dependencies, the resource and its downstream dependencies, or the resource with all dependencies. This menu is enabled for all executable nodes.
+
+#### Results subtab
+The Results subtab displays the most recent Preview results in tabular format. 
+
+#### Compiled code subtab
+The Compiled Code Tab shows the compiled code generated by the most recent compile invocation triggered by the Compile button.
+
+#### Lineage subtab
+The Lineage subtab shows the lineage for the active model in the File Editor. By default, this menu will display 2 degrees of lineage in both directions (`2+model_name+2`). This can be edited to display any desired selection syntax. 
+
+#### Minimap
+A Minimap (code outline) gives you a high-level overview of your source code, which is useful for quick navigation and code understanding. A file's minimap is displayed on the upper-right side of the editor. To quickly jump to different sections of your file, click the shaded area.
+
+#### Editor Command Palette
+The Editor Command Palette contains text editing shortvuts, and lists the associated keyboard shortcut for each action where applicable. This menu can be accessed using F1 or by right clicking in the File Editor menu and selecting Command Palette.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-command-palette.jpg" width="100%" title="Click F1 to access the Editor Command Palette menu for editor shortcuts"/>
+
+#### Editor tab menu
+The file tab menu contains helpful options for interacting with open editor tabs. You can access the menu by right-clicking on any open tab. 
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-tab-menu.jpg" width="100%" title="Right click a tab to view the Editor tab menu options"/>
+
+#### Git Diff View
+After clicking on a file in the **Changes** section in the **Version Control Menu**, the changed file will open with the Git Diff view. The editor will display the previous version on the left, and the highlighted, in-line changes made on the right. 
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-git-diff-view.jpg" width="100%" title="The Git Diff View displays the previous version on the left and the changes made on the right of the Editor"/>
+
+#### Markdown Preview subtab
```

Comment:
We've called the `Interaction bar` the `Console` and the `subtab` the `consoleTab` in our code, calling the bottom portion the `console`. 

Which do you prefer -- Console and Console Tab or Interaction Bar and Subtab? (I lean console!)

Reply:
I would too! Didn't know the existing verbiage!